src/pages/settings/organization/Members.vue

```
@@ -356,18 +382,72 @@ async function didCancel() {
+                  if (!selectedUserToDelegateAdmin.value) {
+                    toast.error(t('please-select-user'))
+                    return false
+                  }
+
+                  // get member from id
+                  const selectedUser = members.value.filter(m => m.id === selectedUserToDelegateAdmin.value)[0]
+                  // set user to super admin
+                  _changeMemberPermission(selectedUser, 'super_admin')
+                  selectedUserToDelegateAdmin.value = null
+                  // get current member
+                  const currentMember = members.value.filter(m => m.uid === main.user?.id )[0]
```

ESLint

[error] There should be no space before this paren.

(style/space-in-parens)